src/SkillService.Data/UserSkillRepository.cs

```diff
@@ -25,11 +25,12 @@ public async Task<List<Guid>> GetUserSkillIdsAsync(Guid userId)
         .Where(us => us.UserId == userId).Select(us => us.SkillId).ToListAsync();
     }
 
-    public async Task<List<DbUserSkill>> GetAsync(Guid userId)
+    public async Task<List<DbSkill>> FindAsync(Guid userId)
```

Comment:
Remove async await from here, you already use await in FindUserSkillCommand (line 31)

Reply:
Done